01/ht/gendos379/01/script.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Makes calculations according to given attributes.
+ * All attributes should be given in separated parenthesis, e.g.:
+ * calc(3)('+')(2)
+ * @param {Number} firstOperand - First number for calculation
+ * @param {String} operator - Arithmetic operator, possible values: '+' / '-' / '/' / '*'
+ * @param {Number} secondOperand - Second number for calculation
+ * @return {Number} - Result of calculation
+ */
+function calc(firstOperand) {
+	return function(operator) {
+  		return function(secondOperand) {
+  			if (operations[operator]) {

```

Comment:
incorrect identation. In js it doesn't influence to logic, but it's still better to keep formatting good

You can use autoformat function from your editor